plugins/module_utils/mso.py

```diff
@@ -916,26 +924,67 @@ def lookup_users(self, users, ignore_not_found_error=False):
             users.append("admin")
 
         ids = []
+        if self.platform == "nd":
+            nd = NDModule(self.module)
+            remote_users = nd.request("/nexus/infra/api/aaa/v4/remoteusers", method="GET")
+            local_users = nd.request("/nexus/infra/api/aaa/v4/localusers", method="GET")
+
         for user in users:
+            user_dict = dict()
             if self.platform == "nd":
-                u = self.get_obj("users", loginID=user, api_version="v2")
+                user_dict = self.get_user_from_list_of_users(user, local_users)
```

Comment:
Was this intentionally designed to lookup remote users if no local users were found?